### ISSUE TITLE
Add specific wording/sentence to a NOTE for emphasis 2.4.11 Focus Appearance (Minimum)

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2463]
Add specific wording/sentence to a NOTE for emphasis 2.4.11 Focus Appearance (Minimum)
https://w3c.github.io/wcag/understanding/focus-appearance-minimum.html

At **"Change of contrast"** there's a text mentioning something NOT excepted:
 
> It is possible to use visual patterns such as strips switching places to disguise a change of focus indicator. This is not considered a visible indicator.

This should be emphasized / highlighted a bit more I think and we can do this by placing it in a NOTE?!

> NOTE
> It is possible to use visual patterns such as strips switching places to disguise a change of focus indicator. This is not considered a visible indicator.

**Comment by alastc** [comment]
Hi Jake,

This was discussed a while back, where there was an issue #1061 which showed a couple of examples.

The decision at the time was to accept that people could game the SC and create something which meets the normative text but disguises the indicator.

The text we added to the note was the extent the group was happy to go to. Adding a note which (isn't normative but) tries to add something as a fail wouldn't work.

I'll close this as there's no new info since we made the last decision.
